docs-site/content/guide/laravel-full-text-search.md

```diff
@@ -0,0 +1,567 @@
+     *
+     * @var array
+     */
+    protected $fillable = [
+        "name",
+        "release_date",
+        "price",
+        "positive",
+        "negative",
+        "app_id",
+        "min_owners",
+        "max_owners",
+        "hltb_single",
+    ];
+
+    /**
+     * The attributes that should be cast to native types.
+     *
+     * @var array
+     */
+    protected $casts = [
+        "release_date" => "datetime",
+        "price" => "float",
+        "positive" => "integer",
+        "negative" => "integer",
+        "app_id" => "integer",
+        "min_owners" => "integer",
+        "max_owners" => "integer",
+        "hltb_single" => "integer",
+    ];
+
+}
+```
+
+</template>
+
+</Tabs>
+
+
+And modify the migration file `database/migrations/**timestamp**_create_games_table.php` to include the necessary columns for the Steam Games dataset:
+
+<Tabs :tabs="['PHP']">
+
+<template v-slot:PHP>
+
+```php{17-25}
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create("games", function (Blueprint $table) {
+            $table->id();
+            $table->timestamps();
+            $table->string("name");
+            $table->timestamp("release_date");
+            $table->float("price");
+            $table->integer("positive");
+            $table->integer("negative");
+            $table->integer("app_id");
+            $table->integer("min_owners");
+            $table->integer("max_owners");
+            $table->integer("hltb_single")->nullable();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        Schema::dropIfExists("games");
+    }
+};
+```
+
+</template>
+
+</Tabs>
+
+You can then run the migration to create the `games` table in the PostgreSQL database:
+
+<Tabs :tabs="['Shell']">
+
+<template v-slot:Shell>
+
+```shell
+./vendor/bin/sail artisan migrate
+
+```
+
+</template>
+
+</Tabs>
+
+We've created a [simple bash script](https://github.com/typesense/showcase-laravel-steam-games-search/blob/master/scripts/sourcedb.sh) to load all the data into the PostgreSQL database. You can add the script in the `scripts` folder and run it in the terminal to populate the `games` table with the Steam Games dataset.
+
+## Step 5: Index the Data Using Laravel Scout
+
+To index the data using Laravel Scout, let's add the `Searchable` trait to the `Game` model, and create a `toSearchableArray` method that returns the indexable data array for the model. This is required only for Models that have a different schema in the database than in Typesense, e.g. Models that include Dates. You can modify the `app/Models/Game.php` file as follows:
+
+<Tabs :tabs="['PHP']">
+
+<template v-slot:PHP>
+
+```php{7,44-55}
```

Comment:
These line highlights seem to highlight a few extra lines. Could you verify?

Reply:
On this case, it doesn't seem to highlight the line itself, rather strange.
![image](https://github.com/typesense/typesense-website/assets/93339170/c8d570a9-34e5-4d50-b6ad-086a5e435bbb)